crates/services/executor/src/executor.rs

```diff
@@ -194,7 +194,7 @@ impl TransactionsSource for OnceTransactionsSource {
         &self,
         _: u64,
         transactions_limit: u16,
-        _: u32,
```

Comment:
I missed that, but the block size should be `u32`.

Reply:
Updated in https://github.com/FuelLabs/fuel-core/pull/2195/commits/e748837be51c4d738912768c0aa33d605444118f